classes/class-mondu-request-wrapper.php

```diff
@@ -0,0 +1,409 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class MonduRequestWrapper {
```

Comment:
Question: Why don't we have a class_exists check here? But we have it in other places.